part-1-ajax/test.js

```diff
@@ -1,76 +1,90 @@
-function getCountryName(code) {
-    $.ajax({
-      //uses the three letter code to get the country's full name
-      }
-    })
-  }
+// function getCountryName(code) {
+//     $.ajax({
+//       //uses the three letter code to get the country's full name
+//       }
+//     })
+//   }
 
-  function parseData(country) {
-    //takes the country data
-    //extracts the name, capital, population, and neighboring countries
-    //iterates through the neighbors array to send off second AJAX request
-  }
-
-  var button = document.getElementsByTagName('button')[0]
-  button.addEventListener('click', function(){
+// function parseData(country) {
+//     takes the country data
+//     extracts the name, capital, population, and neighboring countries
+//     iterates through the neighbors array to send off second AJAX request
+// }
 
+var button = document.getElementsByTagName('button')[0]
```

Comment:
good job. but in the future try to slim down your code. for example, the success callbacks for each ajax request could be defined elsewhere for better readability.